### PR TITLE
T3T1: emulator

### PR DESCRIPTION
**Pull request by mmilata**
Most of the changes are actually in tests. Changed also the unrelated tests to switch to internal model names, see #3226.

### CODE REVIEW

File: tests/device_tests/test_msg_change_wipe_code_t2.py [hunk 1]
```diff
@@ -118,10 +126,15 @@ def test_set_wipe_code_to_pin(client: Client):
     _ensure_unlocked(client, PIN4)
 
     with client:
-        br_amount = 7 if client.debug.model == "T" else 8
+        br_count = {
+            models.T2T1: 7,
+            models.T2B1: 8,
+            models.T3T1: 7,
+        }[client.model]
+        print(br_count)
```
Comment (obrusvit) [inline tests/device_tests/test_msg_change_wipe_code_t2.py:134]: forgotten `print`?

File: tests/device_tests/test_sdcard.py [hunk 2]
```diff
@@ -113,7 +113,10 @@ def input_flow_change_pin_format():
         client.debug.input("1234")
 
         yield  # SD card problem
-        TR.assert_in(layout().text_content(), "sd_card__unplug_and_insert_correct")
+        if client.model is models.T2T1:
```
Comment (obrusvit) [inline tests/device_tests/test_sdcard.py:116]: not exactly correct as this is dependent also on the date of manufacturing. But it's true for the emulator, so let's keep it.

File: tests/device_tests/test_firmware_hash.py [hunk 3]
```diff
@@ -2,21 +2,22 @@
...
 from trezorlib.debuglink import TrezorClientDebugLink as Client
 
 FIRMWARE_LENGTHS = {
-    "1": 7 * 128 * 1024 + 64 * 1024,
-    "T": 13 * 128 * 1024,
-    "Safe 3": 13 * 128 * 1024,
+    models.T1B1: 7 * 128 * 1024 + 64 * 1024,
+    models.T2T1: 13 * 128 * 1024,
+    models.T2B1: 13 * 128 * 1024,
+    models.T3T1: 13 * 128 * 1024,
```
Comment (obrusvit) [inline tests/device_tests/test_firmware_hash.py:12]: Not directly related to this PR, but how is this value determined? And can we link some source here?

**Review by obrusvit — COMMENTED** [review]
Some comments.

File: tests/device_tests/test_sdcard.py (hunk 2 above)
Comment (mmilata) [reply tests/device_tests/test_sdcard.py]: Good point, 0079dc0045391b8e615caf2d162e447320a0296c now allows both variants of the text.

File: tests/device_tests/test_firmware_hash.py (hunk 3 above)
Comment (mmilata) [reply tests/device_tests/test_firmware_hash.py]: Thinking about it, it should probably correspond to the overall size of `FIRMWARE_AREA`, and thus is actually wrong for T3T1. Should be fixed in 0079dc0045391b8e615caf2d162e447320a0296c

File: tests/click_tests/test_lock.py [hunk 4]
```diff
@@ -32,11 +34,21 @@
 def test_hold_to_lock(device_handler: "BackgroundDeviceHandler"):
     debug = device_handler.debuglink()
 
-    short_duration = 1000 if debug.model == "T" else 500
-    lock_duration = 3500 if debug.model == "T" else 1200
+    short_duration = {
```
[the remark this answers is not in the document]
Comment (mmilata) [reply tests/click_tests/test_lock.py:37]: Not sure, I'm afraid these were determined experimentally.

File: tests/click_tests/test_tutorial.py [hunk 5]
```diff
@@ -31,7 +31,7 @@
 
 
 # TR-only
-pytestmark = [pytest.mark.skip_t1, pytest.mark.skip_t2]
+pytestmark = [pytest.mark.skip_t1, pytest.mark.skip_t2, pytest.mark.skip_t3t1]
```
[the remark this answers is not in the document]
Comment (mmilata) [reply tests/click_tests/test_tutorial.py]: ok, let's do it: 1c771de921609843f06df175931572ccc5b6f399

File: core/src/apps/debug/__init__.py [hunk 6]
```diff
@@ -171,7 +171,11 @@ async def dispatch_DebugLinkDecision(msg: DebugLinkDecision) -> None:
         debug_events.last_event += 1
 
         # TT click on specific coordinates, with possible hold
```
[the remark this answers is not in the document]
Comment (mmilata) [reply core/src/apps/debug/__init__.py]: fixed in 0079dc0045391b8e615caf2d162e447320a0296c

File: core/translations/cs.json [hunk 7]
```diff
@@ -13,6 +13,13 @@
       "3_FONT_MONO": "font_robotomono_medium_20_cs.json",
       "4_FONT_BIG": null,
       "5_FONT_DEMIBOLD": "font_tthoves_demibold_21_cs.json"
+    },
+    "T3T1": {
```
[the remark this answers is not in the document]
Comment (mmilata) [reply core/translations/cs.json:17]: For the sake of this PR, let's keep everything the same as on T2T1, including fonts. We'll change them with introduction of `model_mercury` UI.

File: tests/ui_tests/common.py [hunk 8]
```diff
@@ -234,17 +234,17 @@ class TestCase:
...
+            models.T2T1: "TT",
+            models.T2B1: "TR",
+        }.get(client.model, client.model.internal_name)
         name, group = _get_test_name_and_group(request.node.nodeid)
         full_language = client.features.language
         assert full_language
         language = full_language[:2]
         return cls(
-            model=f"T{model_name}",
+            model=model_name,
```
[the remark this answers is not in the document]
Comment (mmilata) [reply tests/ui_tests/common.py:247]: Yes, this is reflected in fixture names. Eventually we should use internal models there as well but I propose to do it in the next wave of _getting rid of model R_ or even in separate PR.

File: tests/device_tests/test_sdcard.py (hunk 2 above)
Comment (obrusvit) [reply tests/device_tests/test_sdcard.py]: I commented there, didn't you duplicate a function?

File: tests/click_tests/test_lock.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (obrusvit) [reply tests/click_tests/test_lock.py:37]: Ok, thanks

**Review by obrusvit — APPROVED** [review]

**Pull request merged**